code/modules/cargo/packs.dm

```diff
@@ -2506,6 +2506,26 @@
 		var/item = pick_n_take(L)
 		new item(C)
 
+/datum/supply_pack/costumes_toys/trekkie
+	name = "Trekkie Costume Crate"
+	desc = "Wear the scrapped concepts for NanoTrasen's jumpsuits, based off popular late-20th century Earth media! While they couldn't be used for the official jumpsuits due to copyright infringement, it's been assured that they can still legally be sold under the label of being 'failed designs'."
```

Comment:
it's not NanoTrasen

Reply:
? all the paperwork has it as NanoTrasen, as well as all the traitor posters